app/api/cardholder/route.ts

```diff
@@ -81,9 +86,24 @@ export async function POST(request: Request) {
 
     const emoji = emojiMap[sigmoji];
     const fullMessage = `Cardholder of ${emoji}: ${message}`;
-    telegramBot.telegram.sendMessage(TELEGRAM_TEST_CHAT_ID, fullMessage, {
-      message_thread_id: 1414,
-    });
+    for (const [chatId, threadId] of TELEGRAM_TEST_CHAT_IDS) {
```

Comment:
Can we include a way to differentiate which chat to send to instead of just sending to both? Like for now just have an ENV variable we can toggle to switch between our test chat, the residency supergroup, and Sigmojis 1?

Reply:
New env var ACTIVE_TELEGRAM_CHATS is a comma separated list (no spaces) of the chat config identifiers we want to send messages to